src/schema/rules/directories.yaml

```diff
@@ -16,10 +16,50 @@
 # The special "root" specifier describes the root of the dataset and only defines subdirectories.
 # No naming convention applies, and the requirement level and opacity would be superfluous.
 #
+study:
+  root:
+    subdirs:
+      - code
+      - docs
+      - derivatives
+      - logs
+      - phenotype
+      - sourcedata
+      - stimuli
+  code:
+    name: code
+    level: optional
+    opaque: true
+  docs:
+    name: docs
+    level: optional
+    opaque: true
+  derivatives:
+    name: derivatives
+    level: optional
+    opaque: true
+  logs:
+    name: logs
+    level: optional
+    opaque: true
+  phenotype:
+    name: phenotype
+    level: optional
+    opaque: false
+  sourcedata:
+    name: sourcedata
+    level: optional
+    opaque: true
+  stimuli:
+    name: stimuli
+    level: optional
+    opaque: true
```

Comment:
Are `phenotype/` and `stimuli/` expected to occur at the top level or inside a `sourcedata/` or `derivatives/` subdirectory?

Reply:
pushed